docs/user_manual.md

```diff
@@ -139,6 +140,33 @@ A current best practice is to save all your job creation parameters in
 version-controlled files in your project directory. This allows you to tie
 your helios job params to changes in your application code.
 
+### Health Checks
+
+When a job is started, Helios can optionally run a health check before registering your service with
+nameless. This prevents the service from receiving traffic before it is ready. After the container
+starts, Helios will execute the health check as follows.
+
+* Begin executing health checks, using exponential backoff from 1 second to 30 seconds.
+* If no health checks succeed after 5 minutes, kill the container. Helios will start it back up and try again.
+* If a health check succeeds
+  * Stop running health checks
+  * Register service with nameless (if job is configured to do so)
```

Comment:
"Register the service in service discovery" Won't always be nameless and this is public so external people won't know about nameless :)